schemas/epi/v1.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
Not really related to this PR, and I might pop up another ticket to look into this (and might need to double-check with the SDS team), but I've noticed that there are some differences between how we're naming these files. For example, in `schemas/brs/` we have only `v2.json` (i.e. with no `v1.json`). But here, we have a `v1.json` but has the `survey_id` that's included in `v2.json` files in other folders.